kategory-kindedj/src/test/java/kategory/kindedj/KategoryShow.kt

```diff
@@ -0,0 +1,7 @@
+package kategory.kindedj
+
+import kategory.HK as HK_K
+
+interface KategoryShow<in F> {
```

Comment:
We actually need the `Show` type class in case you want to leave this in non test sources and add an issue to create the instances down the road.